packages/context/test/unit/binding.unit.ts

```diff
@@ -52,6 +52,28 @@ describe('Binding', () => {
       expect(binding.tags.has('t1')).to.be.true();
       expect(binding.tags.has('t2')).to.be.true();
     });
+
+    it('tags the binding with rest args', () => {
+      binding.tag('t1', ['t2', 't3'], 't4');
+      expect(binding.tags.has('t1')).to.be.true();
+      expect(binding.tags.has('t2')).to.be.true();
+      expect(binding.tags.has('t3')).to.be.true();
+      expect(binding.tags.has('t4')).to.be.true();
```

Comment:
When one of the assertion fails, the error message is rather unhelpful: _expected false to be true_.

Please rewrite the test to assert on the array of tag keys, something along the following lines:

```ts
expect(binding.tags.keys()).toContainUnordered(['t1', 't2', 't3', 't4']);
```

Reply:
Fixed.